src/decoderWorker.js

```diff
@@ -30,10 +30,31 @@ var OggOpusDecoder = function( config ){
   this.decoderSampleRate = config['decoderSampleRate'] || 48000;
   this.outputBufferSampleRate = config['outputBufferSampleRate'] || 48000;
   this.resampleQuality = config['resampleQuality'] || 3;
+  this.rawPacket = config['rawPacket'] || false;
   this.outputBuffers = [];
+  if (this.rawPacket){ this.inited = false };
 };
 
 OggOpusDecoder.prototype.decode = function( typedArray ) {
+  if (this.rawPacket) {
+    if (!this.inited) {
```

Comment:
Naming, could be confusing because this is only used for rawPackets. Perhaps change the name to `rawPacketInitialized`